app/src/main/java/tech/relaycorp/gateway/domain/StoreParcel.kt

```diff
@@ -53,7 +53,7 @@ class StoreParcel
         }
 
         if (isParcelAlreadyCollected(parcel)) {
-            return Result.DuplicatedParcel(parcel)
+            return Result.CollectedParcel(parcel)
```

Comment:
Tangentially related to this PR, but I thought this described the outcome more accurately given the condition it depends on (`isParcelAlreadyCollected`).